Backoffice/NavigationPanel/Strategies/GeneralNodesPanelStrategy.php

```diff
@@ -11,7 +11,7 @@
  */
 class GeneralNodesPanelStrategy extends AbstractNavigationPanelStrategy
 {
-    const ROLE_ACCESS_GENERAL_NODE = 'ROLE_ACCESS_GENERAL_NODE';
```

Comment:
I think that you should keep and deprecate this constant.

It might be used somewhere